### PR TITLE
Changed main file name

### PR DESCRIPTION
**Pull request by Calmynt**
A test pull request

### CODE REVIEW

**Comment by Calmynt** [comment]
> it suxx

:(

File: Cargo.lock [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+[[package]]
+name = "BendeRS"
```
Comment (ShinySaana) [inline Cargo.lock]:
```suggestion
name = "GenderBendeRS"
```

Comment (ShinySaana) [reply Cargo.lock]: @Calmynt What do you think?

Comment (Calmynt) [reply Cargo.lock]: saana

**Pull request closed without merging**